File: Lista fabio (soluções)/ESTRUTURA REPETIÇÃO 04/Atividade L/questao08.py
'''
Leia N , LimiteSuperior e LimiteInferior e escreva todos os múltiplos de N entre os limites lidos
'''

def main():
    numero = int(input("numero: "))
    limiteInf = int(input("limite inferior: "))
    limiteSup = int(input("limite superior: "))

    while limiteInf <= limiteSup:
        if eh_multiplo(numero, limiteInf):
            print(limiteInf)
        limiteInf += 1

def eh_multiplo(n,l):
    if l % n == 0:
        return True
    else:
        return False

main()

File: Lista fabio (soluções)/ESTRUTURA REPETIÇÃO FOR/Atividade P/questao08.js
// Leia N , LimiteSuperior e LimiteInferior e escreva todos os múltiplos de N entre os limites lidos.

import { get_number } from "./utils/io_utils.js"

function main(){
    let N = get_number("N value: ")
    let limiteInferior = get_number("Limite inferior: ")
    let limiteSuperior = get_number("Limite superior: ")

    for (let c = limiteInferior; c <= limiteSuperior; c++){
        if (c % N == 0){
            console.log(c)
        }
    }
}

main()